packages/runtime-core/src/scheduler.ts

```diff
@@ -162,7 +162,7 @@ export function flushPreFlushCbs(
     }
     for (
       preFlushIndex = 0;
-      preFlushIndex < activePreFlushCbs.length;
+      preFlushIndex < (activePreFlushCbs?.length ?? 0);
```

Comment:
We intentionally avoid using optional chaining in the codebase because we target ES2016 and TS will transpile it to something much more verbose.